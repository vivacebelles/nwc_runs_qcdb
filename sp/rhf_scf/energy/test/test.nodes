===2018-09-27-04:15:01 PM run nodes===
compute-0-13 
