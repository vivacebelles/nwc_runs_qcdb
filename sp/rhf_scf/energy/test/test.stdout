 argument  1 = test.nw
 


============================== echo of input deck ==============================
echo

memory 400 mb
geometry noautosym nocenter units Angstrom
O     0.000000000000    0.000000000000   -0.065638538099
H     0.000000000000   -0.757480611647    0.520865616174
H    -0.000000000000    0.757480611647    0.520865616174
end

basis spherical
* library cc-pvdz
end

scf
end

task scf energy
================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2014
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = compute-0-13.local
    program         = nwchem
    date            = Thu Sep 27 16:15:03 2018

    compiled        = Mon_Oct_12_17:43:00_2015
    source          = /home/weverett/nwchem-build/nwchem-6.5
    nwchem branch   = 6.5
    nwchem revision = 26243
    ga revision     = 10506
    input           = test.nw
    prefix          = test.
    data base       = ./test.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107194 doubles =    100.0 Mbytes
    stack    =   13107199 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428793 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)


          ------
          auto-z
          ------
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000    -0.06563854
    2 H                    1.0000     0.00000000    -0.75748061     0.52086562
    3 H                    1.0000     0.00000000     0.75748061     0.52086562
 
      Atomic Mass 
      ----------- 
 
      O                 15.994910
      H                  1.007825
 

 Effective nuclear repulsion energy (a.u.)       9.1873342402

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.9762757725
 


                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles
 
      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95800
    2 Stretch                  1     3                       0.95800
    3 Bend                     2     1     3               104.50000
 
 
            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000    -0.06563854
 H                     0.00000000    -0.75748061     0.52086562
 H                     0.00000000     0.75748061     0.52086562
 
 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.81036  |     0.95800
    3 H                |   1 O                |     1.81036  |     0.95800
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.50
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/weverett/nwchem/6.5-all/data/libraries/>
  


 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.17200000E+04  0.000710
  1 S  1.75900000E+03  0.005470
  1 S  4.00800000E+02  0.027837
  1 S  1.13700000E+02  0.104800
  1 S  3.70300000E+01  0.283062
  1 S  1.32700000E+01  0.448719
  1 S  5.02500000E+00  0.270952
  1 S  1.01300000E+00  0.015458
 
  2 S  1.17200000E+04 -0.000160
  2 S  1.75900000E+03 -0.001263
  2 S  4.00800000E+02 -0.006267
  2 S  1.13700000E+02 -0.025716
  2 S  3.70300000E+01 -0.070924
  2 S  1.32700000E+01 -0.165411
  2 S  5.02500000E+00 -0.116955
  2 S  1.01300000E+00  0.557368
 
  3 S  3.02300000E-01  1.000000
 
  4 P  1.77000000E+01  0.043018
  4 P  3.85400000E+00  0.228913
  4 P  1.04600000E+00  0.508728
 
  5 P  2.75300000E-01  1.000000
 
  6 D  1.18500000E+00  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30100000E+01  0.019685
  1 S  1.96200000E+00  0.137977
  1 S  4.44600000E-01  0.478148
 
  2 S  1.22000000E-01  1.000000
 
  3 P  7.27000000E-01  1.000000
 


 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       14   3s2p1d
 H                          cc-pvdz                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------
 
 

  ao basis        = "ao basis"
  functions       =    24
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./test.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       14   3s2p1d
 H                          cc-pvdz                  3        5   2s1p



 Forming initial guess at       0.5s

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.76222910
 
      Non-variational initial energy
      ------------------------------

 Total energy =     -75.924655
 1-e energy   =    -121.762071
 2-e energy   =      36.650083
 HOMO         =      -0.469202
 LUMO         =       0.091173
 

 Starting SCF solution at       0.7s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 3.081D+03 #integrals = 2.434D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./test.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 409206
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -75.9918556859  8.32D-01  3.67D-01      0.6
                 2      -76.0244802806  1.74D-01  7.82D-02      0.6
                 3      -76.0267444212  1.47D-02  6.37D-03      0.9
                 4      -76.0267607373  3.42D-05  1.90D-05      0.9


       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.026760737256
      One-electron energy =   -123.137554344130
      Two-electron energy =     37.923459366709
 Nuclear repulsion energy =      9.187334240165

        Time for solution =      0.3s


             Final eigenvalues
             -----------------

              1      
    1  -20.5506
    2   -1.3363
    3   -0.6989
    4   -0.5665
    5   -0.4931
    6    0.1854
    7    0.2561
    8    0.7887
    9    0.8537
   10    1.1636
   11    1.2004
   12    1.2534
   13    1.4446
   14    1.4761
   15    1.6745
 
                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.336319D+00
              MO Center=  7.8D-17, -1.1D-15,  1.1D-01, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.442964  1 O  s                  3      0.376217  1 O  s          
    15      0.193306  2 H  s                 20      0.193306  3 H  s          
 
 Vector    3  Occ=2.000000D+00  E=-6.988706D-01
              MO Center= -7.1D-17, -3.9D-16,  1.6D-01, r^2= 7.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.489886  1 O  py                15     -0.327839  2 H  s          
    20      0.327839  3 H  s                  8      0.222097  1 O  py         
 
 Vector    4  Occ=2.000000D+00  E=-5.664603D-01
              MO Center=  9.7D-17,  1.3D-15, -1.1D-01, r^2= 6.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.545401  1 O  pz                 9      0.365380  1 O  pz         
     3     -0.349607  1 O  s                 15      0.206415  2 H  s          
    20      0.206415  3 H  s                  2     -0.150354  1 O  s          
 
 Vector    5  Occ=2.000000D+00  E=-4.930984D-01
              MO Center= -2.4D-16,  1.5D-16, -4.2D-02, r^2= 6.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.631196  1 O  px                 7      0.495684  1 O  px         
 
 Vector    6  Occ=0.000000D+00  E= 1.854409D-01
              MO Center=  1.2D-16, -4.1D-14,  6.6D-01, r^2= 3.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.002033  1 O  s                 16     -0.828996  2 H  s          
    21     -0.828996  3 H  s                  9      0.337289  1 O  pz         
     6      0.190768  1 O  pz         
 
 Vector    7  Occ=0.000000D+00  E= 2.561440D-01
              MO Center= -9.3D-18,  3.2D-14,  6.7D-01, r^2= 3.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      1.443127  2 H  s                 21     -1.443127  3 H  s          
     8      0.670968  1 O  py                 5      0.283379  1 O  py         
 
 Vector    8  Occ=0.000000D+00  E= 7.887331D-01
              MO Center= -8.8D-17, -5.3D-14,  3.0D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.944260  2 H  s                 20     -0.944260  3 H  s          
    16     -0.687203  2 H  s                 21      0.687203  3 H  s          
     8      0.459993  1 O  py                 5      0.268128  1 O  py         
    19      0.152696  2 H  pz                24     -0.152696  3 H  pz         
 
 Vector    9  Occ=0.000000D+00  E= 8.536828D-01
              MO Center=  1.2D-16,  5.6D-14,  5.2D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.788060  2 H  s                 20      0.788060  3 H  s          
    16     -0.548167  2 H  s                 21     -0.548167  3 H  s          
     6     -0.329231  1 O  pz                 3      0.319884  1 O  s          
    18     -0.295536  2 H  py                23      0.295536  3 H  py         
     2     -0.256009  1 O  s          
 
 Vector   10  Occ=0.000000D+00  E= 1.163570D+00
              MO Center=  1.1D-16, -6.5D-15, -8.7D-02, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.278199  1 O  pz                 6     -0.754618  1 O  pz         
     3      0.747372  1 O  s                 15     -0.545742  2 H  s          
    20     -0.545742  3 H  s                 19      0.251214  2 H  pz         
    24      0.251214  3 H  pz         
 
 Vector   11  Occ=0.000000D+00  E= 1.200374D+00
              MO Center=  1.9D-16, -1.6D-15, -6.1D-02, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7     -1.024643  1 O  px                 4      0.967831  1 O  px         
 
 Vector   12  Occ=0.000000D+00  E= 1.253358D+00
              MO Center= -9.2D-16,  3.7D-15, -7.2D-02, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.763511  1 O  py                16      0.824408  2 H  s          
    21     -0.824408  3 H  s                  5     -0.734381  1 O  py         
    15      0.379020  2 H  s                 20     -0.379020  3 H  s          
    18      0.302795  2 H  py                23      0.302795  3 H  py         
    19     -0.186409  2 H  pz                24      0.186409  3 H  pz         
 
 Vector   13  Occ=0.000000D+00  E= 1.444597D+00
              MO Center= -3.3D-16,  3.2D-16,  1.1D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.740266  1 O  pz                19     -0.545122  2 H  pz         
    24     -0.545122  3 H  pz                 2      0.529227  1 O  s          
     3     -0.505882  1 O  s                 15     -0.333461  2 H  s          
    20     -0.333461  3 H  s                 18     -0.329782  2 H  py         
    23      0.329782  3 H  py                16      0.209279  2 H  s          
 
 Vector   14  Occ=0.000000D+00  E= 1.476109D+00
              MO Center=  1.2D-15,  4.4D-15,  4.8D-01, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.685555  2 H  px                22     -0.685555  3 H  px         
 
 Vector   15  Occ=0.000000D+00  E= 1.674463D+00
              MO Center= -1.0D-15, -2.7D-15,  3.4D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.766952  2 H  px                22      0.766952  3 H  px         
     7     -0.634059  1 O  px                13     -0.160829  1 O  d  1       
 

 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000003

 moments of inertia (a.u.)
 ------------------
           6.328975696429           0.000000000000           0.000000000000
           0.000000000000           2.198920681254           0.000000000000
           0.000000000000           0.000000000000           4.130055015175
 
  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     8.31   2.00  0.83  0.82  2.82  1.82  0.01
    2 H    1     0.85   0.69  0.07  0.09
    3 H    1     0.85   0.69  0.07  0.09
 
       Multipole analysis of the density wrt the origin
       ------------------------------------------------
 
     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000     10.000000
 
     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.809374      0.000000      0.976276
 
     2   2 0 0     -5.230344      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -3.063584      0.000000      4.097988
     2   0 1 1     -0.000000      0.000000      0.000000
     2   0 0 2     -4.217882      0.000000      2.060752
 

 Parallel integral file used       4 records with       0 large values


 Task  times  cpu:        0.6s     wall:        1.3s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  205      205     1094      505      982        0        0       60     
number of processes/call 1.29e+00 1.21e+00 1.12e+00 0.00e+00 0.00e+00
bytes total:             1.09e+06 6.56e+05 1.71e+05 0.00e+00 0.00e+00 4.80e+02
bytes remote:            1.72e+05 6.73e+04 7.72e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 37536 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        14
	current total bytes		         0	         0
	maximum total bytes		   1061552	  22509656
	maximum total K-bytes		      1062	     22510
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
        S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov,
    F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
         P.-D. Fan, A. Fonari, R. J. Harrison, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.7s     wall:        1.6s
